src/System.Runtime.Extensions/src/System/Environment.WinRT.cs

```diff
@@ -0,0 +1,138 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation.Metadata;
+using Windows.Storage;
+using System.IO;
+
+namespace System
+{
+    public static partial class Environment
+    {
+        private static string GetFolderPathCore(SpecialFolder folder, SpecialFolderOption option)
+        {
+            // For testing we'll fall back if the needed APIs aren't present.
+            //
+            // We're not honoring the special folder options (noverify/create) for a few reasons. One, most of the
+            // folders always exist (e.g. it is moot). Two, most locations are inaccessible from an appcontainer
+            // currently - making it impossible to answer the question of existence or create if necessary. Thirdly,
```

Comment:
So most of the paths we're handing out will result in exceptions when callers try to use them?